published v3.0.0 does not contain dist folder

While upgrading to `v3.0.0` I saw our build failing  with error:

```js
Error: Cannot find module 'node_modules/mini-html-webpack-plugin/dist/index.js'. Please verify that the package.json has a valid "main" entry
```

Upon inspecting the `node_modules` I found that it does not contains the `dist` folder.
Given below is list of files inside `node_modules/mini-html-webpack-plugin` for `v3.0.0`.

```sh
list node_modules/mini-html-webpack-plugin

├── License.md
├── README.md
└── package.json

0 directories, 3 files
```

Could it be that the latest release did not publish `dist` folder ?

--- a/package.json
+++ b/package.json
@@ -9,6 +9,7 @@
 		"dist/index.d.ts"
 	],
 	"scripts": {
+		"prepublishOnly": "npm run build",
 		"build": "rimraf ./dist && tsc",
 		"watch": "tsc --watch",
 		"pretest": "npm run lint",

